opta/commands/apply.py

```diff
@@ -197,7 +197,7 @@ def _apply(
 
```

Comment:
Let's just remove this warning for now

Reply:
... why? There is a good reason we added it, and removing it is jumpy